examples/fastapi/advanced.py

```
@@ -0,0 +1,432 @@
+"""
+FastAPI + pyhub-llm 고급 통합 예제
+
+이 파일은 인증, Rate Limiting, 고급 서비스 등을 포함한
+완전한 프로덕션 레벨의 FastAPI 애플리케이션 예제입니다.
+"""
+
+import os
+import asyncio
+from typing import List, Optional, Dict, Any
```

Flake8 (7.2.0)

[error] 'typing.List' imported but unused

(F401)